src/ctelnet.h

```diff
@@ -139,6 +139,20 @@ const char MSDP_TABLE_CLOSE = 4;
 const char MSDP_ARRAY_OPEN = 5;
 const char MSDP_ARRAY_CLOSE = 6;
 
+// https://tintin.mudhalla.net/protocols/mtts/
+const int MTTS_STD_ANSI = 1; // Client supports all common ANSI color codes.
+const int MTTS_STD_VT100 = 2; // Client supports all common VT100 codes.
+const int MTTS_STD_UTF_8 = 4; // Client is using UTF-8 character encoding.
+const int MTTS_STD_256_COLORS = 8; // Client supports all 256 color codes.
+const int MTTS_STD_MOUSE_TRACKING = 16; // Client supports xterm mouse tracking.
+const int MTTS_STD_OSC_COLOR_PALETTE = 32; // Client supports the OSC color palette.
+const int MTTS_STD_SCREEN_READER = 64; // Client is using a screen reader.
+const int MTTS_STD_PROXY = 128; // Client is a proxy allowing different users to connect from the same IP address.
```

Comment:
Does this use of the word "proxy" mean the same thing as what Mudlet does with a proxy connection? :thinking:

Reply:
We'll remove until we know more. This would be good to add back in if we sent the user's IPADDRESS field as well, which is useful for game admins to block people who abuse proxy connections.